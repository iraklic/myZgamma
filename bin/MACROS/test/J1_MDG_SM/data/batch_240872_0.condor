000 (240872.000.000) 12/15 14:18:43 Job submitted from host: <131.225.191.196:59123>
...
001 (240872.000.000) 12/15 14:19:03 Job executing on host: <131.225.189.168:37149>
...
006 (240872.000.000) 12/15 14:19:11 Image size of job updated: 81624
...
005 (240872.000.000) 12/15 14:27:32 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:08:16, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:08:16, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	54568  -  Run Bytes Sent By Job
	1420804  -  Run Bytes Received By Job
	54568  -  Total Bytes Sent By Job
	1420804  -  Total Bytes Received By Job
...
