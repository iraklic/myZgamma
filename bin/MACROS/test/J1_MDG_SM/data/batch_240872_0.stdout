
CMSSW on Condor

started at Thu Dec 15 14:19:03 CST 2011

parameter set:
CONDOR_CLUSTER: 240872
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_data.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_240872_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_240872_0.log 2>&1\n

finished at Thu Dec 15 14:27:32 CST 2011
